selfswab/utils.py

```diff
@@ -26,7 +26,9 @@ def get_next_unique_contact_id():
 
 
 def is_barcode_format_valid(barcode):
-    matches = re.findall(r"^(CP159600)|(00[0-9]|0[0-9][0-9]|[0-9][0-9][0-9])$", barcode)
+    matches = re.findall(
+        r"^(CP1596[0-9][0-9])|(00[0-9]|0[0-9][0-9]|[0-9][0-9][0-9])$", barcode
+    )
```

Comment:
I know it wasn't changed in this PR, but the `(CP159600)|...` will mean that `CP159612` will be valid, but also `001`, or `012`, or `123`. That doesn't seem correct, that you need the whole `CP159612` to match, but just the string `012` also matches.

We can also simplify this a bit, `(00[0-9]|0[0-9][0-9]|[0-9][0-9][0-9])` is the same as `([0-9]{3})`